src/flask/api_setup.py

```diff
@@ -37,5 +39,6 @@ def setup_api(app):
     app.register_blueprint(product_ingredient_blueprint)
     app.register_blueprint(product_blueprint)
     app.register_blueprint(order_status_history_blueprint)
+    app.register_blueprint(oauth2_blueprint)
```

Comment:
Move this to be the first route to register.

Reply:
done.